src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -145,12 +162,54 @@ public void getExistingBrapiData(List<BrAPIImport> importRows, Program program)
     public Map<String, ImportPreviewStatistics> process(
             List<BrAPIImport> importRows,
             Map<Integer, PendingImport> mappedBrAPIImport,
+            Table data,
             Program program,
             User user,
             boolean commit) throws ValidatorException, MissingRequiredInfoException {
 
         ValidationErrors validationErrors = new ValidationErrors();
 
+        // Get dynamic phenotype columns for processing
+        List<Column<?>> phenotypeCols = fileMappingUtil.getDynamicColumns(data, "ExperimentsTemplateMap");
```

Comment:
Is there a constant that's defined anywhere for the `ExperimentsTemplateMap`? If not, can that be added?

Reply:
No, I'll add one. Pushed changes.